tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -57,10 +60,16 @@ spec:
         COMPONENTS_LENGTH=$(jq '.components |length' ${SNAPSHOT_PATH})
         for (( COMPONENTS_INDEX=0; COMPONENTS_INDEX<COMPONENTS_LENGTH; COMPONENTS_INDEX++ )); do
 
-            reference=$(jq -r ".components[${COMPONENTS_INDEX}].containerImage" ${SNAPSHOT_PATH})
-            manifest_digest="${reference#*@}"
+            referenceContainerImage=$(jq -r ".components[${COMPONENTS_INDEX}].containerImage" ${SNAPSHOT_PATH})
+
+            reference=$(jq -r ".components[${COMPONENTS_INDEX}].repository" ${SNAPSHOT_PATH}):$(params.commonTag)
+            manifest_digest="${referenceContainerImage#*@}"
+
+            echo "Creating InternalRequest to sign image:"
+            echo "- reference=${reference}"
```

Comment:
are these just for debugging? Or do we want them in here permanently?

Reply:
I wanted to see this permanently. I foresee this being beneficial in the future.